### ISSUE TITLE
Add new linkml validaton plugin to make-squeaky clean

### ISSUE THREAD

**Issue by samobermiller** [microbiomedata/nmdc-schema#2705]
We may want `make-squeaky clean test all` to be validating with the new linkml plugin that checks storage_unit slots (#2648). Otherwise the test files will pass/fail incorrectly during github actions (example `substances_volume` in #2689)

**Comment by pkalita-lbl** [comment]
I think the source of the issue isn't the test, but the example file itself. The file `src/data/valid/Database-interleaved.yaml` is not technically a valid YAML file. There are duplicate keys strewn all about it. For instance there are _three_ top-level `workflow_execution_set` keys. Because of [known issue](https://github.com/yaml/pyyaml/issues/165) in the de-facto Python YAML parsing library, this situation doesn't raise a parsing error but instead the last key overwrites previous ones. So in effect the test isn't even seeing the object with the `substances_volume` slot.

I would recommend using [`yamllint`](https://yamllint.readthedocs.io/en/stable/index.html) to identify issues with that example file first. For example:
```
yamllint -d "{rules: {key-duplicates: enable}}" src/data/valid/Database-interleaved.yaml
```

**Comment by samobermiller** [comment]
I can try that out. I would say it looks like the same now incorrect parameter is used in https://github.com/microbiomedata/nmdc-schema/blob/bb3c2b8a2dd648af97422a86618d04cb8af68e9c/src/data/valid/ChemicalConversionProcess-digest.yaml#L25, which doesn't look like it has a duplicate key problem

**Comment by pkalita-lbl** [comment]
Oh awesome, the test isn't even running in GitHub Actions because the `test` Makefile target ultimately invokes `unittest`: https://github.com/microbiomedata/nmdc-schema/blob/f7699ca15462c94a05ef6ab8df7f8f2539a5b669/Makefile#L153 

And the [dumb-dumb who wrote the test](https://github.com/microbiomedata/nmdc-schema/commit/cd933e10ee4f44a1f2774e1bd7c5da4aaf498888) wrote it assuming a `pytest` runner. My bad.
